src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -12,117 +11,25 @@ const createNewStep = (context, step, args) => {
   ) ? 'host.docker.internal'
     : `remoter-server-${experimentId}.${config.pipelineNamespace}.svc.cluster.local`;
 
-  const task = JSON.stringify({
+  const task = {
     experimentId,
     taskName,
     config: processingConfig[taskName] || {},
     server: remoterServer,
-  });
-
-  const stepHash = crypto.randomBytes(5).toString('hex');
-
-  if (config.clusterEnv === 'development') {
-    return {
-      ...step,
-      Type: 'Task',
-      Resource: 'arn:aws:states:::lambda:invoke',
-      // TODO: fix this, add some of old output
-      // see https://docs.aws.amazon.com/step-functions/latest/dg/input-output-resultpath.html
-      ResultPath: null,
-      Parameters: {
-        FunctionName: `arn:aws:lambda:eu-west-1:${accountId}:function:local-container-launcher`,
-        Payload: {
-          image: 'biomage-remoter-client',
-          name: 'pipeline-remoter-client',
-          task,
-          ...perSample && { 'sampleUuid.$': '$.sampleUuid' },
-          detached: false,
-        },
-      },
-      Catch: [
-        {
-          ErrorEquals: ['States.ALL'],
-          ResultPath: '$.error-info',
-          Next: step.XNextOnCatch || step.Next,
-        },
-      ],
-    };
-  }
-
+  };
 
   return {
     ...step,
     Type: 'Task',
-    Comment: 'Attempts to create a Kubernetes Job for the pipeline server. Will swallow a 409 (already exists) error.',
-    Resource: 'arn:aws:states:::eks:runJob.sync',
```

Comment:
if this is no longer going to be an eks job, you should remember to delete the pipeline can spawn role for k8s in pipeline repo.

Reply:
Again, we still have a kubernetes job step for spawning the actual server, so we can keep it.

Reply:
good point, ok